adapters/fiducia-adapter/src/main/java/de/adorsys/xs2a/adapter/fiducia/model/FiduciaPeriodicPaymentInitiationXmlPart2StandingorderTypeJson.java

```diff
@@ -0,0 +1,79 @@
+package de.adorsys.xs2a.adapter.fiducia.model;
+
+import de.adorsys.xs2a.adapter.api.model.FrequencyCode;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+public class FiduciaPeriodicPaymentInitiationXmlPart2StandingorderTypeJson {
```

Comment:
maybe order should start with a capital `O`, what do you think?

Reply:
It's a copy of a generated class that has lowercase 'o' as defined in the openapi spec https://github.com/adorsys/xs2a-adapter/blob/develop/xs2a-adapter-codegen/src/main/resources/psd2-api%201.3.6%2020200306v1.yaml#L6528. Unfortunately the spec is quite sloppy in this regard. In my opinion it's not worthwhile to attempt to correct these spelling issues in the model copies.

Reply:
agree